src/routes/savoringYourChildModule/savoringYourChildModule.js

```diff
@@ -9,67 +9,108 @@ import Share from '../../components/Share'
 import FAQ from '../../components/FAQ'
 
 type Props = {
-  title: string,
-  path: string,
-  typeform: string,
-  faq: [],
-  testimonials: [],
-  user: Object,
-  typeformUserId: '',
+  title?: string,
+  path?: string,
+  typeform?: string,
+  faq?: [],
+  testimonials?: [],
+  user?: Object,
+  typeformUserId?: '',
 };
 
-const savoringYourChildSectionForm = ({
-  title, path, typeform, faq, testimonials, user, typeformUserId,
-}:
-Props) => (
-  <div>
-    <div className="container">
-      <div className="row">
-        <div className="col-md-2 col-xs-12">
-          <Share path={path} title={title} />
-        </div>
-        <div className="col-md-8 col-xs-12">
-          <div className="mainheading">
-            <h1 className="posttitle">{title}</h1>
-          </div>
-          <div>
-            <div>User typeform ID: {typeformUserId},</div>
-            <div>User ID {user._id},</div>
+class savoringYourChildSectionForm extends React.Component {
+  static defaultProps = {
+    title: '',
+    path: '',
+    typeform: '',
+    faq: '',
+    testimonials: '',
+    user: '',
+    typeformUserId: '',
+  }
+
+  state = {
```

Comment:
why use state and not just a `const = {` ? only use state if you are using it (i.e. `setState` etc)